packages/create/templates/hello-express/src/main.ts

```diff
@@ -0,0 +1,16 @@
+import { app } from './operations';
+import { DBOS } from '@dbos-inc/dbos-sdk';
+
+async function main() {
+  await DBOS.launch({expressApp: app});
+
+  const PORT = DBOS.runtimeConfig?.port ?? 9000;
```

Comment:
Nit: The default port for Express.js is 3000. Why do we use 9000 here?

Reply:
What port does the DBOS Cloud forward?

Reply:
DBOS Cloud forwards to 3000 for node.js apps and 8000 for Python apps.